README.md

```diff
@@ -342,7 +342,7 @@ If `src` responds to `call` its return value will be used as the scripttag's sou
 AfterAuthenticate Job
 ---------------------
 
-If your app needs to perform specific actions after it is installed ShopifyApp can queue or run a job of your choosing (note that we already provide support for automatically creating Webhooks and Scripttags). To configure the after authenticate job update your initializer as follows:
+If your app needs to perform specific actions after the shop is authenticated successfully (i.e every time a new session is created), ShopifyApp can queue or run a job of your choosing (note that we already provide support for automatically creating Webhooks and Scripttags). To configure the after authenticate job update your initializer as follows:
```

Comment:
Thanks for your pull request @NafaaBout.

As Shopify stores can have multiple users, we suggest "If your app needs to perform specific actions after the **user** is authenticated successfully (i.e every time a new session is created), ShopifyApp can queue..." instead.